check_perl fails

### Describe the bug

https://github.com/openfoodfacts/openfoodfacts-server/pull/7124 appeared to be failing and the error didn't seem to be related to the PR. 

The error:
```
[Fri Jul 22 18:46:24 2022] login.pl: Variable "$cookie" is not imported at cgi/login.pl line 52.
[Fri Jul 22 18:46:24 2022] login.pl: 	(Did you mean &cookie instead?)
Status: 500
Content-type: text/html

<h1>Software error:</h1>
<pre>Global symbol &quot;$cookie&quot; requires explicit package name (did you forget to declare &quot;my $cookie&quot;?) at cgi/login.pl line 52.
cgi/login.pl had compilation errors.
</pre>
<p>
For help, please send mail to this site's webmaster, giving this error message 
and the time and date of the error.

</p>
[Fri Jul 22 18:46:24 2022] login.pl: Global symbol "$cookie" requires explicit package name (did you forget to declare "my $cookie"?) at cgi/login.pl line 52.
[Fri Jul 22 18:46:24 2022] login.pl: cgi/login.pl had compilation errors.
if [ -f cgi/nutrients.pl ]; then perl -c -CS -Ilib cgi/nutrients.pl; else true; fi
make: *** [Makefile:225: cgi/login.pl] Error 255
```

So I reran the test on the main branch and it still failed.

### To Reproduce

1. Go to the main branch
2. Run `make check_perl`

Or make a PR and it should supposedly fail

### Expected behavior

For `make check_perl` to run successfully, with no errors.

### Screenshots

_No response_

### Additional context

_No response_

### Type of device

Other

### Browser version

_No response_

### Number of products impacted

_No response_

### Time per product

_No response_

--- a/cgi/login.pl
+++ b/cgi/login.pl
@@ -49,7 +49,7 @@ $template_data_ref->{redirect} = $redirect;
 if (defined $User_id) {
 	my $loc = $redirect || $formatted_subdomain;
 	$r->headers_out->set(Location => $loc);
-	$r->err_headers_out->add('Set-Cookie' => $cookie);
+	$r->err_headers_out->add('Set-Cookie' => $request_ref->{cookie});
 	$r->status(302);
 	return Apache2::Const::OK;
 }

--- a/cgi/product_image.pl
+++ b/cgi/product_image.pl
@@ -174,7 +174,7 @@ my $html;
 process_template('product_image.tt.html', $template_data_ref, \$html) or $html = '';
 $html .= "<p>" . $tt->error() . "</p>";
 
-$request_ref->{title} = $title;
+$request_ref->{title} = $alt;
 $request_ref->{content_ref} = \$html;
 $request_ref->{full_width} = 0;
 display_page($request_ref);

--- a/cgi/reset_password.pl
+++ b/cgi/reset_password.pl
@@ -199,7 +199,7 @@ elsif ($action eq 'process') {
 
 process_template('web/pages/reset_password/reset_password.tt.html', $template_data_ref, \$html) or $html = "<p>" . $tt->error() . "</p>";
 
-$request_ref->{title} = $title;
+$request_ref->{title} = $Lang{'reset_password'}{$lang};
 $request_ref->{content_ref} = \$html;
 display_page($request_ref);
 

--- a/cgi/session.pl
+++ b/cgi/session.pl
@@ -78,8 +78,8 @@ if (defined $User_id) {
 
 		$r->err_headers_out->add('Set-Cookie' => $request_ref->{cookie});
 		$r->headers_out->set(Location => "$url");
-		$r->status(301);
-		return 301;
+		$r->status(302);
+		return 302;
 	}
 }
 

